From 8cb2ed4145b6c742dafb9892c6f96c93421c9656 Mon Sep 17 00:00:00 2001
From: Tomas Hancock <tom.hancock@hp.com>
Date: Wed, 12 Oct 2011 16:08:10 +0100
Subject: [PATCH] Fixes LP Bug#872276 - small typo in error message

Add a missing s to a % formatted print statement
Add my name to Authors file

(cherry picked from commit 6aacf2388a03a138502d1ed05dce6c3bfb0502cc)

Change-Id: Ib086fb9bee6f55a3bd89f3108269050c6eb0f8b9
---
 Authors    |    1 +
 bin/glance |    2 +-
 2 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/Authors b/Authors
index 673e8a3..3d2ec33 100644
diff --git a/bin/glance b/bin/glance
index 0cbe4a2..166b695 100755
--- a/bin/glance
+++ b/bin/glance
@@ -939,7 +939,7 @@ option is given, <MEMBER> will be able to further share the image."""
         c.add_member(image_id, member_id, options.can_share)
     else:
         print "Dry run. We would have done the following:"
-        print ('Add "%(member_id)" to membership of image %(image_id)s'
+        print ('Add "%(member_id)s" to membership of image %(image_id)s'
                % locals())
         if options.can_share:
             print "New member would have been able to further share image."
-- 
1.7.6.5

